test/react-native/features/support/env.rb

```diff
@@ -1,4 +1,6 @@
 BeforeAll do
-  Maze.config.android_app_files_directory = '/data/local/tmp'
+  if Maze.config.farm == :bb
```

Comment:
BrowserStack doesn't allow pushing files to `/data/local/tmp` so we only override this on BitBar